tenable_io/helpers/workbench.py

```diff
@@ -0,0 +1,417 @@
+import tempfile
+
+from tenable_io.api.workbenches import WorkbenchesApi
+from tenable_io.parser.workbenches import WorkbenchParser
+from tenable_io.util import wait_until
+
+
+class WorkbenchHelper(object):
```

Comment:
Should we add a download helper function?

For example, something like this mirrors the scan download function:
```
    def download(self,
                 path,
                 format=WorkbenchesApi.FORMAT_NESSUS,
                 report=WorkbenchesApi.REPORT_VULNERABILITIES,
                 chapter=WorkbenchesApi.CHAPTER_VULN_BY_ASSET,
                 start_date=None,
                 date_range=None,
                 filters=None,
                 filter_search_type=None,
                 minimum_vuln_info=None,
                 plugin_id=None,
                 asset_id=None,
                 file_open_mode='wb'):
        """Download a workbench.

        :param path: The file path to save the export to.
        :param format: The file format, one of FORMAT_CSV, FORMAT_HTML, FORMAT_NESSUS, FORMAT_PDF. Defaults to
            FORMAT_NESSUS.
        :param report: The type of workbench report, one of REPORT_VULNERABILITIES. Defaults to REPORT_VULNERABILITIES.
        :param chapter: Chapter to include, one of CHAPTER_DIFF, CHAPTER_EXEC_SUMMARY, CHAPTER_VULN_BY_ASSET,
            CHAPTER_VULN_BY_PLUGIN, CHAPTER_VULN_HOSTS_SUMMARY. Defaults to CHAPTER_VULN_BY_ASSET.
        :param start_date: The date (in unixtime) at which the exported results should begin to be included, default to
            None which implies today.
        :param date_range: The number of days of data prior to and including start_date that should be returned. default
            to None which implies data for all dates is returned.
        :param filters: An list containing filters to apply to the exported scan report, default to None.
        :param filter_search_type: The type of search to be used, one of FILTER_TYPE_AND, FILTER_TYPE_OR, default to
            None.
        :param minimum_vuln_info: When True, only a minimal subset of scan details will be returned for each result,
            excluding plugin attributes. In this case, only plugin_output and vulnerability_state fields are always
            returned; first_found, last_found and last_fixed are also returned if possible. Default to None.
        :param plugin_id: Restrict the export data to only vulnerabilities found by the plugin with this id, default to
            None.
        :param asset_id: Restrict the export data to only findings the asset with this id. Note that this id is a UUID,
            default to None.
        :param file_open_mode: The open mode to the file output. Default to "wb".
        """

        file_id = self._client.workbenches_api.export_request(format=format,
                                                              report=report,
                                                              chapter=chapter,
                                                              start_date=start_date,
                                                              date_range=date_range,
                                                              filters=filters,
                                                              filter_search_type=filter_search_type,
                                                              minimum_vuln_info=minimum_vuln_info,
                                                              plugin_id=plugin_id,
                                                              asset_id=asset_id)

        wait_until(lambda: self._client.workbenches_api.export_status(file_id) == WorkbenchesApi.STATUS_EXPORT_READY)

        iter_content = self._client.workbenches_api.export_download(file_id)
        with open(path, file_open_mode) as fd:
            for chunk in iter_content:
                fd.write(chunk)

```